addons/website_slides/static/src/scss/slides_slide_fullscreen.scss

```
@@ -107,6 +107,10 @@
             }
         }
     }
+    .o_wslides_additional_resources{
+        padding-left: 20px;
+        margin-bottom: 0;
+    }
```

```suggestion
    .o_wslides_additional_resources {
        margin-bottom: 0;
        padding-left: 20px;
    }
```

Also, doesn't this apply the padding both to the ul and li? Is it on purpose even for the left padding?

---

yes, left padding is inherited by li from ul elements, so I decided to change it from there.

---

I meant that it applies to `<ul>` too. Anyway, if the result is as expected, it's good for me :)